packages/cli/src/utils.ts

```diff
@@ -310,7 +310,12 @@ const getAndroidDetails = async (
     min_sdk: parseInt(minSdk?.[1] ?? "0", 10),
     version_code: parseInt(versionCode?.[1] ?? "0", 10),
     version: versionName?.[1] ?? "0",
-    permissions: permissions.flatMap(permission => permission[1]),
+    permissions: permissions.flatMap(permission => {
+      if (permission[1] === undefined)
+        return permission[3]
+      else
+        return permission[1] + " maxSdkVersion=" + permission[2]
```

Comment:
I'd recommend we add a comment here noting that this format will change in the future, with a link to the GitHub issue.